Reqnroll/Formatters/RuntimeSupport/TestRowPickleMapper.cs

```diff
@@ -0,0 +1,50 @@
+using Io.Cucumber.Messages.Types;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Reqnroll.Formatters.RuntimeSupport;
+
+public static class TestRowPickleMapper
+{
+    public static object ComputeHash(string featureName, string scenarioOutlineName, IEnumerable<string> tags, IEnumerable<string> rowValues)
+    {
+        var tagsList = tags ?? Enumerable.Empty<string>();
+        var rowValuesList = rowValues ?? Enumerable.Empty<string>();
+        var v = $"{featureName}|{scenarioOutlineName}|{string.Join("|", tagsList)}|{string.Join("|", rowValuesList)}";
+        return v.GetHashCode();
```

Comment:
HashCodes aren't likely to overlap but they are not unique.
Is this a problem here?